products/sle/main.pm

```diff
@@ -1316,6 +1316,13 @@ elsif (get_var('HPC')) {
         loadtest 'hpc/munge_slave'  if (check_var('HPC', 'munge_slave'));
     }
 }
+elsif (get_var('ROOTONLY')) {
+    load_boot_tests();
+    load_inst_tests();
+    load_reboot_tests();
+    loadtest "x11/xterm";
+    loadtest "x11/gnome_terminal";
```

Comment:
I have discussed this with Yifan and we would think about what a "root user" only system admin usually does for his daily work. May be, working on a terminal.

And many console based case will require 'user-console', even consoletest_setup will change user between normal user and root, we think it's not necessary to update all these cases only for this scenario.

So only two cases(xterm and gnome_terminal) are included by this case.